tests/admin_inlines/tests.py

```diff
@@ -361,6 +361,24 @@ def test_inline_nonauto_noneditable_pk(self):
             html=True
         )
 
+    def test_non_autofield_primary_key(self):
+        """
+        Ensure that the inline form work when the inner model inherits a model
```

Comment:
Try to state the expected behavior rather than "Ensure that" or "Test that" since all tests do that. Ticket references are only needed for obscure issues that can't be explained in the docstring. I'm not sure that's the case here.